.dev/compare_branches.R

```diff
@@ -104,10 +117,13 @@ if (interactive()) {
     # typed arguments get cast even when missing, probably to NA
     if (isTRUE(is.na(params[[opt]]) || params[[opt]] == "")) params[[opt]] <- NULL
   }
+  if (isTRUE(is.na(params$base_branch) || params$base_branch == "")) params$base_branch <- "master"
```

Comment:
use | here? are we sure length 1?

Reply:
Comes from `optparse`. I'd be surprised to see anything but a length-1 character vector.

Reply:
NB we lint `if (a | b)` 😉

Reply:
not if (isTRUE(a | b)) though 😎

Reply:
Anyway. I don't see any optparse documentation that hints at the ability to parse more than length-1 arguments, and we know our argument list only contains length-1 arguments. I reckon even the `isTRUE()` isn't strictly necessary.

Reply:
ok anyway this is a dev script... doesn't need to be perfectly robust.